routing/control_tower.go

```diff
@@ -14,25 +16,35 @@ import (
 // restarts. Payments are transitioned through various payment states, and the
 // ControlTower interface provides access to driving the state transitions.
 type ControlTower interface {
-	// InitPayment atomically moves the payment into the InFlight state.
 	// This method checks that no suceeded payment exist for this payment
 	// hash.
 	InitPayment(lntypes.Hash, *channeldb.PaymentCreationInfo) error
 
 	// RegisterAttempt atomically records the provided HTLCAttemptInfo.
+	//
+	// TODO: should it check that the total value is not exceeded? Use case
```

Comment:
i think we talked about doing this in the control tower?

Reply:
Comment applies to mpp sending. Removed here.